modules/storage-persistent-storage-lifecycle.adoc

```diff
@@ -57,9 +57,18 @@ PVs by including `persistentVolumeClaim` in the Pod's volumes block.
 ifdef::openshift-origin,openshift-enterprise[]
 
 [id="pvcprotection_{context}"]
-== PVC protection
+== Storage Object in Use Protection
 
-PVC protection is enabled by default.
+The Storage Object in Use Protection feature ensures that PVCs in active use by a Pod and PVs that are bound to PVCs are not removed from the system, as this may result in data loss.
```

Comment:
You need to go up and expand PV and PVC on first use.
s/may/can

Reply:
PV and PVC are expanded in first use of the [assembly](https://docs.openshift.com/container-platform/4.2/storage/understanding-persistent-storage.html#provisioning_understanding-persistent-storage). Elsewhere, h2s are using the initialisms. LMK if that's not correct.

Reply:
If each module is meant to be standalone, there's no guarantee that first use will be elsewhere.